### PR TITLE
Include code alerts in api responses

### PR DESCRIPTION
**Pull request by nicomwong**
### Testing
Tested organizations and repositories with fake code_alerts; both worked
Did not test with more than 1 organization

### Changes
Organizations response: add "code_alerts_exist" field
Repositories response:   add "code_alerts" field

### CODE REVIEW

File: app/controllers/api/repos_controller.rb [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
-    def check_org
-      if !@organization
-        render json: { message: 'No organizations were found' }, status: :not_found
-      end
-    end
-end
+class Api::ReposController < ApplicationController
+    before_action :get_orgs, :check_org
+    def index      
+      repos = @organization.repositories
```
Comment (nicomwong) [inline app/controllers/api/repos_controller.rb:4]: only changed index action; don't know why git shows full file diff

File: app/controllers/api/repos_controller.rb [hunk 2]
```diff
@@ -1,18 +1,26 @@
...
-    end
-
-    def check_org
-      if !@organization
-        render json: { message: 'No organizations were found' }, status: :not_found
-      end
-    end
-end
+class Api::ReposController < ApplicationController
+    before_action :get_orgs, :check_org
```
Comment (nicomwong) [inline app/controllers/api/repos_controller.rb:2]: only changed index action; don't know why git shows full file diff

File: app/controllers/api/repos_controller.rb [hunk 3]
```diff
@@ -1,18 +1,26 @@
...
+    def index      
+      repos = @organization.repositories
+      repos_json = JSON.parse(repos.to_json())
+
+      # add code_alerts field to repositories json
+      repos.zip(repos_json).each { |repo, repo_json|
+        repo_json["code_alerts"] = repo.code_alerts
```
Comment (chrisyang365) [inline app/controllers/api/repos_controller.rb:9]: per your suggestion in Discord, change this to display boolean `code_alerts_exist` field just like you did for orgs

Comment (chrisyang365) [inline app/controllers/api/repos_controller.rb]: per your suggestion in Discord, change this to display boolean `code_alerts_exist` field just like you did for orgs

**Review by chrisyang365 — CHANGES_REQUESTED** [review]

**Review by chrisyang365 — APPROVED** [review]

**Pull request merged**